src/winrtble/manager.rs

```diff
@@ -35,7 +35,7 @@ impl api::Manager for Manager {
         Ok(radios
             .into_iter()
             .filter(|radio| radio.Kind() == Ok(RadioKind::Bluetooth))
-            .map(|_| Adapter::new())
+            .map(|radio| Adapter::new(radio.clone()))
```

Comment:
Why does this need to clone the radio?

Reply:
you are right, https://github.com/deviceplug/btleplug/pull/366/commits/3e090847b7002e1a454bd6944e03455ab957ccbd removed clone plus added `radio.StateChanged` error handler